src/dist/replication/test/simple_kv/simple_kv.server.impl.h

```diff
@@ -90,6 +90,8 @@ class simple_kv_service_impl : public application::simple_kv_service
 
     virtual void query_app_envs(/*out*/ std::map<std::string, std::string> &envs) {}
 
+    virtual void set_partition_version(int32_t partition_version) {}
```

Comment:
```
class replication_app_base {
  virtual void set_partition_version(int32_t partition_version) {}
};
```

You can implement `replication_app_base::set_partition_version` as non-pure virtual function, so that
simple_kv.server.impl.h don't need changes, mock_utils.h as well.

Reply:
I used to write as non-pure virtual function, I update it as liwei's comment. Besides, `mock_utils.h` need this function in further pull request.

Reply:
OK. mock_utils.h can override this function if it needs, but simplekv doesn't.

Reply:
update this function to non-pure virtual function